### ISSUE TITLE
Applications added to metaprotocol level 

### ISSUE THREAD

**Issue by dmtrjsg** [Joystream/atlas#3400]
## Context
- https://github.com/Joystream/joystream/issues/4307

Runtime will be adding apps ID and other properties to the metaprotocol level and extending  Atlas would need to introduce the relevant changes for thsi to happen.

## Scope
- [ ]  Update content dir update extrinsics to include app meta data

**Comment by WRadoslaw** [comment]
Blocked by [PR](https://github.com/Joystream/joystream/pull/4504)

**Comment by dmtrjsg** [comment]
superseded by 
- https://github.com/Joystream/atlas/issues/3519
